core/src/main/java/com/linecorp/armeria/server/ServiceConfigBuilder.java

```diff
@@ -217,6 +219,13 @@ public ServiceConfigBuilder setHeaders(
         return this;
     }
 
+    @Override
+    public ServiceConfigSetters errorHandler(ServiceErrorHandler serviceErrorHandler) {
+        requireNonNull(serviceErrorHandler, "serviceErrorHandler");
+        this.serviceErrorHandler = serviceErrorHandler;
```

Comment:
Could you also add the field to the `toString()` method?
Or I think we can just remove `toString()` method in builder classes. /cc @line/dx